src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -1,67 +1,66 @@
 
   // Text input
   --#{$text-input-group}__text-input--PaddingTop: var(--#{$pf-global}--spacer--form-element);
-  --#{$text-input-group}__text-input--PaddingRight: var(--#{$pf-global}--spacer--sm);
+  --#{$text-input-group}__text-input--PaddingRight: var(--pf-t--global--spacer--sm);
   --#{$text-input-group}__text-input--PaddingBottom: var(--#{$pf-global}--spacer--form-element);
-  --#{$text-input-group}__text-input--PaddingLeft: var(--#{$pf-global}--spacer--sm);
+  --#{$text-input-group}__text-input--PaddingLeft: var(--pf-t--global--spacer--sm);
   --#{$text-input-group}__text-input--MinWidth: 12ch;
-  --#{$text-input-group}__text-input--m-hint--Color: var(--#{$pf-global}--Color--200); // matches placeholder text color
-  --#{$text-input-group}__text-input--placeholder--Color: var(--#{$pf-global}--Color--200);
+  --#{$text-input-group}__text-input--m-hint--Color: var(--pf-t--global--text--color--default);
+  --#{$text-input-group}__text-input--placeholder--Color: var(--pf-t--global--text--color--placeholder);
   --#{$text-input-group}__text-input--BackgroundColor: transparent;
 
   // Icon
-  --#{$text-input-group}__icon--Left: var(--#{$pf-global}--spacer--sm);
-  --#{$text-input-group}__icon--Color: var(--#{$pf-global}--icon--Color--dark);
+  --#{$text-input-group}__icon--Left: var(--pf-t--global--spacer--sm);
+  --#{$text-input-group}__icon--Color: var(--pf-t--global--icon--color--regular);
   --#{$text-input-group}__icon--TranslateY: -50%;
 
   // Utilities
-  --#{$text-input-group}__utilities--child--MarginLeft: var(--#{$pf-global}--spacer--xs);
-  --#{$text-input-group}__utilities--c-button--PaddingRight: var(--#{$pf-global}--spacer--xs);
-  --#{$text-input-group}__utilities--c-button--PaddingLeft: var(--#{$pf-global}--spacer--xs);
+  --#{$text-input-group}__utilities--child--MarginLeft: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}__utilities--c-button--PaddingRight: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}__utilities--c-button--PaddingLeft: var(--pf-t--global--spacer--xs);
 
   // input disabled style
-  --#{$text-input-group}--m-disabled--Color: var(--#{$pf-global}--disabled-color--100);
-  --#{$text-input-group}--m-disabled--BackgroundColor: var(--#{$pf-global}--disabled-color--300);
+  --#{$text-input-group}--m-disabled--Color: var(--pf-t--global--text--color--on-disabled);
+  --#{$text-input-group}--m-disabled--BackgroundColor: var(--pf-t--global--background--color--disabled--default);
+}
 
+.#{$text-input-group} {
   position: relative;
   display: flex;
+  align-items: flex-start;
   width: 100%; // when child of flex layout, take up space
   color: var(--#{$text-input-group}--Color, inherit);
   background-color: var(--#{$text-input-group}--BackgroundColor);
+  border: var(--#{$text-input-group}--BorderWidth) solid var(--#{$text-input-group}--BorderColor);
+  border-radius: var(--#{$text-input-group}--BorderRadius);
```

Comment:
These will need to go on the (absolutely positioned) pseudo element for this div since this element needs to match the height of other control-type stuff, and the height of control things is defined by the text size, line-height, and block padding. The borders on this div will make this component taller by 2px (top + bottom borders) than other control things. Absolutely positioned things do not create layout/space, so they won't change the size of anything.